server/views/pages/barcode/create-new-contact-by-prison-number.njk

```diff
@@ -5,7 +5,7 @@
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/select/macro.njk" import govukSelect %}
 
-{% set pageTitle = applicationName + " - Create contact" %}
+{% set pageTitle = applicationName + " - Create contact number given" %}
```

Comment:
You didnt get a rational explanation as to what these new titles mean then? 🤣

Reply:
Nah. I was blindsided by "SEO" - obviously above my paygrade 😁